MonoGame.Framework/BoundingFrustum.cs

```diff
@@ -4,103 +4,178 @@
 
 using System;
 using System.Diagnostics;
-using System.Text;
 
 namespace Microsoft.Xna.Framework
 {
+    /// <summary>
+    /// Defines a frustum for intersection operations.

```

Comment:
I would try to get a little better definition about what a frustum is here and how it is build from a matrix.  Not something huge, but something a little more.